spec/inputs/exec_spec.rb

```diff
@@ -10,6 +10,31 @@
     expect {input.register}.to_not raise_error
   end
 
+  context "when operating normally" do
+    let(:input) { LogStash::Plugin.lookup("input", "exec").new("command" => "uptime", "interval" => 0) }
```

Comment:
This command wont work on windows, we can replace it with **ls**, which should work. 
Which should work on Linux, mac and windows powershell.